### ISSUE TITLE
fetchTrades returns GMT as localized timestamp in JS

### ISSUE THREAD

**Issue by D-Nice** [ccxt-dev/ccxt#325]
Only tested with Poloniex and fetchTrades, but the converted timestamp is wrong. The library appears to assume that the time Poloniex is returning is my local timezone, and not GMT. The time fetched (which is already GMT) is converted to GMT, and makes orders appear as though they happened in the future.

**Comment by kroitor** [comment]
Hi, @D-Nice! What's your programming language version of ccxt?

**Comment by kroitor** [comment]
Please note, that all timestamps and datetimes are UTC!

```Python
import ccxt
poloniex = ccxt.poloniex()
trades = poloniex.fetchTrades('BTC/USDT')
print('Current UTC time:', poloniex.iso8601(poloniex.milliseconds()))
print('Last trade was on UTC time:', trades[0]['datetime'])
```

The little python test above should show your system's UTC time, and will show the UTC datetime of the last trade on Poloniex, so they should be very close to each other.

```
mbp:ccxt igorkroitor$ python
Python 2.7.13 (default, Jun  5 2017, 10:04:07) 
[GCC 4.2.1 Compatible Apple LLVM 8.1.0 (clang-802.0.42)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import ccxt
>>> poloniex = ccxt.poloniex()
>>> trades = poloniex.fetchTrades('BTC/USDT')
>>> print('Current UTC time:', poloniex.iso8601(poloniex.milliseconds()))
('Current UTC time:', '2017-10-17T00:34:40.000Z')
>>> print('Last trade was on:', trades[0]['datetime'])
('Last trade was on:', '2017-10-17T00:34:20.000Z')
>>> 
```

**Comment by D-Nice** [comment]
I'm using node. I did try your test, but the python version appears to have issues working at all. Regardless, node is my personal preference anyways.

I've cloned your test script for node 8+:

```js
const ccxt = require('ccxt')

const polo = new ccxt['poloniex']()

console.log(polo.iso8601(polo.milliseconds()))

;(async() => {
  console.log((await polo.fetchTrades('BTC/USDT'))[0].datetime)
})()
```

here's the output

```
2017-10-17T02:11:02.877Z
2017-10-17T06:11:02.000Z
```

And the issue I am talking about.

Thanks for your time and help!

**Comment by kroitor** [comment]
@D-Nice thx a lot for the snippet!

Can you please try the same for Bitfinex now? 

```JavaScript
const ccxt = require('ccxt')
const ex = new ccxt.bitfinex()
console.log(ex.iso8601(ex.milliseconds()))
;(async() => { console.log(ex.id, (await ex.fetchTrades('BTC/USD'))[0].datetime) })()
```

Looks like a Polo issue to me... A difference of 3 hours, where the same functions return no difference for Bitfinex UTC time... For Kraken it seems to be 1 hour. If that is the case, we will have to add some corrections for that. Should be UTC, really. Or there may be a bug inside the library, will test for that as well.

>  I did try your test, but the python version appears to have issues working at all. 

If you could post an error log or something, it would be awesome. I tried that test by copying it and pasting into launched python interpreter shell inside the library directory and it worked for me. If it doesn't work for you, then, maybe we should do some debugging on that as well, we would appreciate your help.

**Comment by kroitor** [comment]
Ok, I did a few more tests and it looks like a bug in parse8601 in js, but I'll fix it quickly. Thx for your report!

**Comment by kroitor** [comment]
Ok, this issue has been fixed as of CCXT version 1.9.156+. Thx for your feedback once again! Let us know if you find any other inconsistencies! Cheers!
